source/gloperate/source/base/GLContextUtils.cpp

```diff
@@ -106,13 +117,43 @@ glbinding::Version GLContextUtils::retrieveVersion()
     return glbinding::Version(major, minor);
 }
 
+GLContextFormat::Profile GLContextUtils::retrieveProfile()
+{
+    assert(0 != glbinding::getCurrentContext());
```

Comment:
Its the same assertion used in (most) other functions of this class. If there is indeed a platform where this is different, there should be a dedicated function to check this, e.g., `assertCurrentContext()`.